airflow/www/jest-setup.js

```diff
@@ -20,6 +20,21 @@
  */
 
 import '@testing-library/jest-dom';
+import axios from 'axios';
+import { setLogger } from 'react-query';
+
+axios.defaults.adapter = require('axios/lib/adapters/http');
+
+axios.interceptors.response.use(
+  (res) => res.data || res,
+);
+
+setLogger({
+  log: console.log,
+  warn: console.warn,
+  // ✅ no more errors on the console
```

Comment:
Maybe a silly question, if they don't end up on the console, where do they end up?

Reply:
This is just for tests. We are intentionally creating API errors. So it doesn't need to be logged.